e2e/cli-e2e/tests/collect.e2e.test.ts

```diff
@@ -49,22 +49,6 @@ describe('CLI collect', () => {
     await cleanTestFolder('tmp/e2e/react-todos-app');
   });
 
-  it('should run ESLint plugin and create report.json', async () => {
-    const { code, stderr } = await executeProcess({
-      command: 'code-pushup',
-      args: ['collect', '--no-progress', '--onlyPlugins=eslint'],
-      cwd: 'examples/react-todos-app',
-    });
-
-    expect(code).toBe(0);
-    expect(stderr).toBe('');
-
-    const report = await readJsonFile('tmp/e2e/react-todos-app/report.json');
-
-    expect(() => reportSchema.parse(report)).not.toThrow();
-    expect(omitVariableReportData(report as Report)).toMatchSnapshot();
```

Comment:
The snapshot from `cli-e2e` should be deleted, as it's now obsolete.

Reply:
done